### DISCUSSION TITLE
replace vanilla bamboo

### DISCUSSION THREAD

**Discussion by NeoForce0** [DynamicTreesTeam/DynamicTrees#854] [Ideas]
I've been testing out the dynamic trees and limited the growth of trees. So wood products take a lot longer to obtain. I noticed when I planted bamboo, that grew way too fast essentially making planks too easy to obtain quickly with that. Even with that, bamboo is different than regular wood so disabling bamboo planks is best way too. Do you have any plans to incorporate dynamic bamboo as well?

**Comment by offbem** [comment]
That's what I was thinking
